spec/spec_helper.rb

```diff
@@ -1,5 +1,6 @@
 if ENV['CI']
   require 'simplecov'
+  SimpleCov.formatter = SimpleCov::Formatter::HTMLFormatter
```

Comment:
@astrozzc can we remove this line, we would have to figure out why this doesn't work inside of the automate_engine repo.

Reply:
Sure, let me remove this line for now.

Reply:
@mkanoor Updated